testcharms/charm-repo/bionic/dispatch/dispatch

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+name=`echo $JUJU_DISPATCH_PATH`
```

Comment:
We haven't merged the dispatch bits yet ;-) 
Perhaps we should remove this file and add it back with that changeset?

Reply:
This file doesn't hurt anything as it won't run currently and it is being used for the embedded commit tests inside of the actions.  So I'd rather leave it.

Reply:
Given the time passed, JUJU_DISPATH_PATH is now in develop